src/mem/ruby/system/GPUCoalescer.hh

```diff
@@ -291,22 +291,25 @@ class GPUCoalescer : public RubyPort
 
     void readCallback(Addr address,
                       MachineType mach,
-                      DataBlock& data);
+                      DataBlock& data,
+                      bool externalHit);
```

Comment:
Have you tested other protocols?  Adding a new parameter without a default seems like it's going to break *any* other protocol which does not add the new parameter.

Reply:
This is a good point ... how do we test this easily?  I had thought the daily/weekly tests would be enough, but maybe not?

Reply:
Oh, scratch that. I didn't notice with was only in GPUCoalescer. A default would still be nice to avoid breaking other GPU protocols.  I don't know what "externalHit" means so I don't know what that should be but either true/false.

Reply:
I think defaulting to false makes sense, as it would at least not make people think something hit when it (maybe) didn't?

Reply:
Sure that makes sense